src/py21cmfast/wrapper.py

```diff
@@ -903,6 +901,71 @@ def calc_zstart_photon_cons():
     return lib.ComputeZstart_PhotonCons()
 
 
+
+def access_photon_nonconservation_data():
```

Comment:
I think the name for this function should be `get_photon_nonconservation_data`... "access" sounds like its being stored globally somewhere and you're just providing a window onto it, however this function actually calculates and returns the data.

This also needs a docstring!

Reply:
Well technically these are now globals as it'll allow easier debugging of the gsl errors. Nevertheless, I have renamed the function and added the docstring